cmake/HPX_SetupBoost.cmake

```diff
@@ -21,6 +21,11 @@ set(Boost_ADDITIONAL_VERSIONS
     "1.58.0" "1.58"
     "1.57.0" "1.57")
 
+set(HPX_BOOST_LOG_LIBS "")
+if(HPX_PARCELPORT_VERBS_WITH_LOGGING OR HPX_PARCELPORT_VERBS_WITH_DEV_MODE)
+  set(HPX_BOOST_LOG_LIBS log log_setup)
+endif()
```

Comment:
The majority of the rdma code was originally in its own mini library and was using boost log. The logging code is only activated when enabled via cmake options and I would not expect anyone other than myself or another developer interested in the verbs parcelport to activate it.
I agree that we should have just one logging options, but this has proven to be my #1 mean of diagnosing problems and leaving it in for the time being is not going to cost anyone. In the long term we should migrate to a single logging infrastructure - but I'd like to postpone this until this is proven to be production stable.